src/structs/lepton_format.rs

```diff
@@ -1273,7 +1284,21 @@ impl LeptonHeader {
         writer.write_u8('M' as u8)?;
         writer.write_u8('S' as u8)?;
         writer.write_u32::<LittleEndian>(lepton_header.len() as u32)?;
-        writer.write_all(&[0; 6])?;
+
+        // write the flags that were used to encode this file
```

Comment:
Why we need this support? Why would we want to ever compress with use_16bit* flags? If it was added for test purposes, I think we need some other approach.